docs/source/guides/setting_config_options.ipynb

```diff
@@ -138,29 +138,27 @@
     "\n",
     "### Email Inference Regex\n",
     "\n",
-    "Woodwork provides the option to infer string columns as the `EmailAddress` logical type if a representative sample of valid (non-missing) rows all match a given regular expression. The `email_inference_regex` config variable allows users to set the regular expression that is used during this matching process. The default regex is `r\"(^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\\.[a-zA-Z0-9-.]+$)\"` (taken from https://emailregex.com/)."
+    "Woodwork provides the option to infer string columns as the `EmailAddress` logical type if a representative sample of valid (non-missing) rows all match a given regular expression. The `email_inference_regex` config variable allows users to set the regular expression that is used during this matching process. The default regex is `r\"(^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\\.[a-zA-Z0-9-.]+$)\"` (taken from https://emailregex.com/).\n",
+    "\n",
+    "### URL Inference Regex\n",
+    "\n",
+    "Woodwork provides the option to infer string columns as the `URL` logical type if a representative sample of valid (non-missing) rows all match a given regular expression. The `url_inference_regex` config variable allows users to set the regular expression that is used during this matching process. The default regex is `r\"(http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\\(\\),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+)\"` (taken from https://urlregex.com/)."
    ]
   }
  ],
  "metadata": {
+  "interpreter": {
+   "hash": "aee8b7b246df8f9039afb4144a1f6fd8d2ca17a180786b69acc140d282b71a49"
+  },
   "kernelspec": {
-   "display_name": "Python 3",
-   "language": "python",
+   "display_name": "Python 3.9.5 64-bit",
    "name": "python3"
   },
   "language_info": {
-   "codemirror_mode": {
-    "name": "ipython",
-    "version": 3
-   },
-   "file_extension": ".py",
-   "mimetype": "text/x-python",
    "name": "python",
-   "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.7.11"
+   "version": ""
```

Comment:
Were these changes automatically included by jupyter?  If so, it might make sense to just leave them out.  Thoughts on this @gsheni @tuethan1999 ?

Reply:
Yeah I think so, I only added the text part.

Reply:
Yeah, I mean they probably don't matter that much.  If anything, I'm mostly just curious what the effect of the "hash" part is.

Reply:
Actually, these things probably don't matter.  We can just include them.